articles/underspecification/index.md

```diff
@@ -0,0 +1,105 @@
+
+On the bright side, there are ways to reduce the impact of underspecification. We cover a handful below.
+
+#### Limiting Model Complexity
+
+A way of addressing underspecification is limiting the complexity of a model. The complexity of modern machine learning models almost guarantees that underspecification will occur. As defined by this [post](https://machinelearningmastery.com/difference-between-a-parameter-and-a-hyperparameter/), a parameter is a configuration variable that is internal to the model and whose value can be approximated from data. A model that has many free parameters encourages underspecification. These parameters allow models to use numerous ways to achieve a prediction. This allows models to blend competent predictions with luck. It is difficult to determine which approach was skill and which was luck.
+
+It is important to understand the proposed real-world application of a model. Consulting domain experts could do this. They could help to fine-tune the scope of the model. It could also be of great importance to work with potential users of the model. They could provide useful insights that could help the model-building process be more objective. A more objective scope would mean fewer free parameters. The model would have a greater focus on its purpose. This would, in turn, keep the model as simple as possible.
+
+It is important to have the complexity of the model under control. The first step is understanding why we choose a specific model architecture for a given task. If we choose an architecture, it helps to consider whether there is a simpler architecture that could produce a similar result. We could also consider combining or reducing variables to limit complexity.
+
+#### Stress Tests
+
+Stress testing is the subjecting of machine learning models to various tests. These tests determine whether model performance is adequate under unfavorable or extreme conditions. Stress tests may help in the detection of underspecification. From this [paper](https://arxiv.org/pdf/2011.03395.pdf), we learn about three types of stress tests. Google researchers used them on their models, with various random seed values. A seed value can be thought of as a parameter that decides the sequence of generated pseudorandom numbers. [Pseudorandomness](https://en.wikipedia.org/wiki/Pseudorandomness) computes the extent to which a sequence of numbers appears to be statistically random, even though a repeatable process produces the numbers.
+
+The three types of stress tests include:
+
+**Stratified Performance Evaluations**. An [inductive bias](http://inductivebias.com/Blog/what-is-inductive-bias/) refers to a set of assumptions that a learner uses to make predictions given never-before encountered inputs. Stratified performance evaluations test whether a predictor encodes inductive biases that give similar performances across different dataset classes. These tests consider sub-groups of observations to identify inequalities.
```

Comment:
The second sentence needs some more explanation. How do the inductive biases get tested for?